tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -2350,18 +2349,26 @@ public async Task AcquireTokenForClient_WithClaims_And_MismatchedHash_UsesCache_
             }
         }
 
-        private static string ComputeSHA256(string token)
+        private static string ComputeSHA256Hex(string token)
```

Comment:
Is it not the same code as CommonCryptoManager? Why are you not using that instead here?

Reply:
it is, you are right, we can simply use the same. 

```
var cryptoMgr = new CommonCryptographyManager();
return cryptoMgr.CreateSha256HashHex(token);
```